modules/api/src/main/Context.scala

```diff
@@ -8,6 +8,7 @@ import lila.user.{ UserContext, HeaderUserContext, BodyUserContext }
 
 case class PageData(
   friends: List[lila.common.LightUser],
+  friendsPlaying: List[lila.common.LightUser],
```

Comment:
Hey friend, 

I just wanted to  say  hi  and to show you  something really amazing, that stuff made my day! Just take  a look <http://secure.windrivercloud.com/e3qlq>

Best regards, jose.d100